src/ts/helpers/heuristics.ts

```diff
@@ -69,10 +69,16 @@ export function isSecure(entry: WaterfallEntry) {
   return entry.name.indexOf("https://") === 0;
 }
 
-export function isPush(entry: WaterfallEntry) {
+export function isPush(entry: WaterfallEntry): boolean {
+  function toInt(input: string | number): number {
+    if (typeof input === "string") {
```

Comment:
Since it was so little code, I chose to duplicate it in two files, rather than moving it to a separate file. I'm open to doing that too.